python/setup.py

```diff
@@ -58,9 +58,31 @@
     conda_include_dir = conda_prefix + '/include'
     conda_lib_dir = conda_prefix + '/lib'
 
+
+class CleanCommand(Command):
+    """Custom clean command to tidy up the project root."""
+    user_options = [('all', None, None), ]
+
+    def initialize_options(self):
+        self.all = None
+
+    def finalize_options(self):
+        pass
+
+    def run(self):
+        os.system('rm -rf build')
```

Comment:
Is there a way you can guarantee you're running these commands from the right directory?  I'm wondering if a user runs something like `python python/setup.py clean` (ie. running it from the cugraph root instead of the python dir), it seems like it could find all the .cpp source files in `<cugraph root>/cpp` and delete those, which would be very bad :)

One suggestion could be to do this:
```
setupFileDir = os.path.dirname(os.path.abspath(__file__))
os.system("cd %s && rm -rf dist" % setupFileDir)
os.system("cd %s && rm -rf dask-worker-space" % setupFileDir)
...
```

Reply:
That would be a costly mistake. fixed incase the user happens to run the script from cugraph_repo_root